docs/modules/ROOT/partials/getting-started/con-registry-rules.adoc

```diff
@@ -27,9 +27,71 @@ NOTE: If no rules are configured for an artifact, the set of currently configure
 
 [id="registry-rules-work"]
 = How rules work
-Each rule has a name and optional configuration information. The registry storage maintains the list of rules for each artifact and the list of global rules. Each rule in the list consists of a name and a set of configuration properties, which are specific to the rule implementation. 
+Each rule has a name and optional configuration information. The registry storage maintains the list of rules for each artifact and the list of global rules. Each rule in the list consists of a name and a set of configuration properties, which are specific to the rule implementation.
 
 A rule is provided with the content of the current version of the artifact (if one exists) and the new version of the artifact being added. The rule implementation returns true or false depending on whether the artifact passes the rule. If not, the registry reports the reason why in an HTTP error response. Some rules might not use the previous version of the content. For example, compatibility rules use previous versions, but syntax or semantic validity rules do not.
 
 .Additional resources
 For more details, see {registry-rule-types} and {registry-rule-maturity-matrix}.
+
+[id="registry-rules-config"]
+= Configuring rules
+You can configure rules individually for each artifact, as well as globally.  {registry} applies the rules configured
+for the specific artifact, but if no rules are configured at that level it applies the globally configured rules.  If
+no global rules are configured then no rules are applied.
+
+== Configuring artifact rules
+You can configure artifact rules using the REST API or the web console.  See the relevant documentation sections
+for more information.
+
+== Configuring global rules
+You can configure global rules in several ways:
+
+* Use the `/rules` operations in the REST API
+* Use the web console
+* Set default global rules using {registry} application properties
+
+.Default global rules
+When deploying {registry} it is possible to configure the application itself to enable or disable global rules.  This
```

Comment:
Suggest "you can configure the application..." for active voice

Reply:
I can make this change, it was just getting repetitive because the previous three sections all begin with "You can configure..."

Despite that do you still recommend the change?

Reply:
Yes, repetition is good, especially for non-native English speakers.

If you go with "When deploying {registry}, you can configure..." it will be slightly different :)